entries/n/nerdwallet.com.json

```
@@ -0,0 +1,17 @@
+{
+  "NerdWallet": {
+    "domain": "nerdwallet.com",
+    "contact": {
+      "facebook": "NerdWallet",
+      "twitter": "NerdWallet"
+    },
+    "regions": [
+      "us",
+      "ca",
+      "gb"
```

There doesn't seem to be any option for UK residents to log in. Probably due to PSD2?
```suggestion
      "ca"
```